docs/installing_postgres.mdx

```diff
@@ -11,25 +11,8 @@ pgxman will support native homebrew installation in the future.
 
 <Note>At this time, only Ubuntu Jammy and Debian Bookworm are supported.</Note>
 
-pgxman assumes you likely already have Postgres installed, and because there are many ways to install
-Postgres, it does not try to install it for you. pgxman will detect your version of Postgres by looking
-for `pg_config`.
-
-If you do not have Postgres installed, we recommend either using the default version included with the distro,
-or use the Postgres Global Developer Group (`pgdg`) repository.
-
-### Included versions
-
-- Debian Bookworm: Postgres 15
-  - `sudo apt install postgresql-15`
-- Ubuntu Jammy: Postgres 14
-  - `sudo apt install postgresql-14`
-
-
-### Other versions
-
-We highly recommend using the Postgres Global Developer Group (PGDG)
-apt repository to install other versions. To add this repository, run:
+pgxman requires using the Postgres Global Developer Group (PGDG) apt repository to install Postgres. To add this
```

Comment:
Maybe link pgdg to https://wiki.postgresql.org/wiki/Apt#:~:text=The%20PostgreSQL%20Global%20Development%20Group,pub%2Frepos%2Fapt%2F.?

Reply:
resolved in updated commit